lib/locket/client.rb

```diff
@@ -7,15 +7,12 @@ class Error < StandardError
     end
 
     def initialize(host:, port:, client_ca_path:, client_cert_path:, client_key_path:, timeout: nil)
-      client_ca = File.open(client_ca_path).read
-      client_key = File.open(client_key_path).read
-      client_cert = File.open(client_cert_path).read
-
-      @service = Models::Locket::Stub.new(
```

Comment:
Changing this here has an impact on the other usage of this client as well. It might be better to only do this (i.e. create new client for each call) in the diego client.

From my point of view in the diego client this is justified as we now also cache the result (active bbs instance). So it's okay if retrieving gets a little bit more expensive (due to the recreation of the locket client) but therefore more resilient to temporary network issues.

Reply:
Yep, another good point. I've switched this around so we remake it only when Diego::Client does a `fetch`.